server/app/views/v1/host_nodes/health.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.id @node.node_id
+json.name @node.name
+json.etcd @etcd_health
```

Comment:
This is what the agent `/etcd/health` RPC returns, so either `{health: etcd.health}` or `{error: error.message}`.